prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/InstallCommandTest.java

```diff
@@ -382,6 +383,12 @@ public void invalidStabilityLevelCausesAnError() throws Exception {
                 .contains(CliMessages.MESSAGES.unknownStabilityLevel("idontexist", InstallCommand.STABILITY_LEVELS).getMessage());
     }
 
+    @Test
+    public void testListProfiles(){
+        int exitCode = commandLine.execute(CliConstants.Commands.INSTALL, CliConstants.LIST_PROFILES);
+        assertEquals(ReturnCodes.SUCCESS, exitCode);
```

Comment:
This test should check that:
1. the server is not installed (i.e. ProvisioningAction is not being called)
2. that the output lists at least the names of available profiles.